packages/clay-management-toolbar/src/ClayManagementToolbar.js

```diff
@@ -220,6 +229,15 @@ ClayManagementToolbar.STATE = {
 	 */
 	id: Config.string(),
 
+	/**
+	 * Flag to indicate if the Info button is needed.
```

Comment:
To be more precise: `Flag to indicate if the Info button should be shown or not.`